.circleci/config.yml

```diff
@@ -496,6 +496,10 @@ jobs:
       - persist_to_workspace:
           root: .
           paths:
+            - "master/dist/*linux_amd64.deb"
```

Comment:
non-blocking: can we persist the dist dir altogether?

Reply:
I'd like to avoid this for now -- every new file adds time to the job, so for now I'd like to keep it to what we need until we need more